src/data/markdown/docs/01 guides/01 Getting started/04 Results Output.md

```diff
@@ -1,77 +1,122 @@
 ---
-title: 'Results Output'
+title: 'Results output'
 excerpt: ''
 ---
 
-## Overview
+By default, when the test runs,  k6 prints runtime information and general results to `stdout`, and when the test finalizes, k6 shows the test result summary.
```

Comment:
I would rewrite this paragraph to instead summarize what is described. Right now it's mostly a repetition of line 12